src/main/python/onnx_systemds/tests/util.py

```diff
@@ -0,0 +1,85 @@
+# Licensed to the Apache Software Foundation (ASF) under one or more
+# contributor license agreements.  See the NOTICE file distributed with
+# this work for additional information regarding copyright ownership.
+# The ASF licenses this file to you under the Apache License, Version 2.0
+# (the "License"); you may not use this file except in compliance with
+# the License.  You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import os
+import subprocess
+import unittest
+
+from onnx_systemds.convert import onnx2systemds
+from onnx_systemds.util import resolve_systemds_root
+
+
+def invoke_systemds(input_file: str, args: [str] = None) -> int:
+    """
+    Runs systemds by running the script in $SYSTEMDS_ROOT_PATH/bin/systemds.sh with the provided input_file,
+    will fail if environment variable SYSTEMDS_ROOT_PATH is not set.
+
+    :param input_file: the dml script to run
+    :param args: additional arguments if needed
+    :return: the return-code of systemds
+    """
+    if args is None:
+        args = []
+
+    systemds_root_path = resolve_systemds_root()
+
+    try:
+        abspath_input = os.path.abspath(input_file)
+        res = subprocess.run([systemds_root_path + "/bin/systemds.sh", abspath_input] + args,
```

Comment:
This is the issue for why it fails now. we changed the name to `/bin/systemds` without the `.sh`.